.cirrus.yml

```diff
@@ -186,6 +192,37 @@ build_task:
     always: *runner_stats
 
 
+build_aarch64_task:
+    alias: 'build_aarch64'
+    name: 'Build for $DISTRO_NV'
+    # Multiarch doesn't depend on buildability in this automation context
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: "$CIRRUS_CRON != 'multiarch'"
+    ec2_instance: &standard_build_ec2_aarch64
+      image: ${FEDORA_AARCH64_AMI_ID}
+      type: t4g.micro
+      region: us-east-1
+      architecture: arm64
+    matrix: *platform_axis
+        # Ref: https://cirrus-ci.org/guide/writing-tasks/#matrix-modification
+        - env: *stdenvars
+              DISTRO_NV: ${FEDORA_NAME_AARCH64}
+              # Not used here, is used in other tasks
+              VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+              CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+              # ID for re-use of build output
+    env:
+        TEST_FLAVOR: build
+    clone_script: *full_clone
+    setup_script: *setup
+    main_script: *main
+    # Cirrus-CI is very slow uploading one file at time, and the repo contains
+    # thousands of files.  Speed this up by archiving into tarball first.
+    repo_prep_script: *repo_prep
+    repo_artifacts: *repo_artifacts
+    always: *runner_stats
+
```

Comment:
Yep, `matrix: *platform_axis` this won't work (and probably isn't what you want since there's no point in a single-axis matrix.

```
...
    ec2_instance: &standard_build_ec2_aarch64
      image: ${FEDORA_AARCH64_AMI_ID}
      type: t4g.micro
      region: us-east-1
      architecture: arm64
    env:
        <<: *stdenvars
        DISTRO_NV: ${FEDORA_NAME_AARCH64}
        TEST_FLAVOR: build
    clone_script: *full_clone
...
```